### ISSUE TITLE
It keeps crashing after 1 song or two

### ISSUE THREAD

**Issue by Baka28** [adlerluiz/ytmdesktop#1452]
### Consent

- [x] I verified that there is no open/closed issue for the same subject.
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

this is the error im getting and i dont really know what this means even if i search it on google so hopefully i can get thelp on this, because i really love using this application. thanks

YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.5
    Electron 29.0.1
    Windows NT 10.0.19045

Name: Error
Message: getaddrinfo ENOTFOUND lh3.googleusercontent.com
Cause: Unknown

Error: getaddrinfo ENOTFOUND lh3.googleusercontent.com
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:118:26)

### Expected Behavior

Before i used an older version. This happened after i updated.

### Steps To Reproduce

nothing really, it just happens

### YTMDesktop

latest

### OS

Windows

### OS Version

10

### Arch

x64

### Installation way

.exe

**Comment by XeroxDev** [comment]
Do you use some kind of proxy/vpn/dns service or system/network wide adblocker? Because this kinda seems like that address is getting blocked.

**Comment by bp88dev** [comment]
I'm having the same issue. No proxy, vpn or anything. This is the first time it's happened. Here's my error below if you guys need it, although it looks exactly the same.

```
YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.5
    Electron 29.0.1
    Windows NT 10.0.22631

Name: Error
Message: getaddrinfo ENOTFOUND lh3.googleusercontent.com
Cause: Unknown

Error: getaddrinfo ENOTFOUND lh3.googleusercontent.com
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:118:26)
```

**Comment by Akiisqt** [comment]
Is this still an issue on the latest branch?

**Comment by bp88dev** [comment]
> Is this still an issue on the latest branch?

I've not been getting the error since I updated to 2.0.6

**Comment by Alipoodle** [comment]
This issues is fixed in 2.0.6.
Basically the fetch for the album art for notifications would send a forced error which can't be fetched
I've chance the method for this and should fix this issue 

If this keeps on happening your notifications will be delayed as it tries to do a lookup for the album art currently.
